api/v1alpha3/virtualmachine_bootstrap_types.go

```diff
@@ -85,6 +85,12 @@ type VirtualMachineBootstrapSpec struct {
 // VirtualMachineBootstrapCloudInitSpec describes the CloudInit configuration
 // used to bootstrap the VM.
 type VirtualMachineBootstrapCloudInitSpec struct {
+	// InstanceID is the cloud-init metadata instance ID.
+	// If omitted, this field defaults to the VM's BiosUUID.
+	//
+	// +optional
+	InstanceID string `json:"instanceID,omitempty"`
```

Comment:
Can both devOps and Admin users specify this field ?

Reply:
Currently, yes, I don't think we called out a need for this field to be admin-only.